tutorials/setting-up-lxd-1604/setting-up-lxd-1604-tutorial.md

```diff
@@ -0,0 +1,211 @@
+
+![screenshot](images/lxd-init-4.png)
+
+The configurator will ask for a few details on how to configure the IPv4 network, specifically:
+
+ * address
+ * CIDR
+ * start/end address for DHCP
+ * maximum number of DHCP clients
+
+We can accept defaults for those questions too.
+
+Finally, answer **Yes** to the question about NAT on the network:
+
+![screenshot](images/lxd-init-5.png)
+
+A similar configuration is available for IPv6 networks, but we can ignore it, and just choose not to set up IPv6.
+
+![screenshot](images/lxd-init-6.png)
+
+
+Done! LXD is now set up and ready to be used.
+
+
+## Launch a container
+Duration: 0:02
+
+It's now time to start using our LXD.
+
+First, let's make sure the client can connect to the daemon, by running `lxc list`
+
+The output should be similar to
+
+```bash
+$ lxc list
+Generating a client certificate. This may take a minute...
+If this is your first time using LXD, you should also run: sudo lxd init
+To start your first container, try: lxc launch ubuntu:16.04
+
++------+-------+------+------+------+-----------+
+| NAME | STATE | IPV4 | IPV6 | TYPE | SNAPSHOTS |
++------+-------+------+------+------+-----------+
+$
+```
+
+which shows there are no running containers.
+
+Let's now launch our first container:
+
+```bash
+$ lxc launch ubuntu:16.04
+Creating stirring-beagle
+Starting stirring-beagle
+```
+
+This will download the official Ubuntu 16.04 LTS (Xenial Xerus) image and
+launch a container with it.
+
+If a container name is not provided (like in this case), a random one will be
+generated.
+
+Let's check that our container is indeed running with
+
+```bash
+$ lxc list
++-----------------+---------+-----------------------+------+------------+-----------+
+|      NAME       |  STATE  |         IPV4          | IPV6 |    TYPE    | SNAPSHOTS |
++-----------------+---------+-----------------------+------+------------+-----------+
+| stirring-beagle | RUNNING | 10.147.177.171 (eth0) |      | PERSISTENT | 0         |
++-----------------+---------+-----------------------+------+------------+-----------+
+```
+
+We can execute commands in the container using `lxc exec`:
+
+```bash
+$ lxc exec stirring-beagle -- ls -la
+total 7
+drwx------  3 root root    5 Jun 29 11:29 .
+drwxr-xr-x 22 root root   22 Jun 19 23:52 ..
+-rw-r--r--  1 root root 3106 Oct 22  2015 .bashrc
+-rw-r--r--  1 root root  148 Aug 17  2015 .profile
+drwx------  2 root root    3 Jun 29 11:29 .ssh
+```
+
+Shell access to the container can be obtained with `lxc exec stirring-beagle /bin/bash`
+
+Note that since we also set up networking, the container has an IPv4 address
+(as shown by `lxc list`) and can be also reached via ssh from the host. This,
+however, requires importing an ssh key in the container first.
+
+
+Once the container is no longer needed, it can be stopped and destroyed:
+
+```bash
+$ lxc stop stirring-beagle
+$ lxc delete stirring-beagle
+```
+
+
+## Finding help
```

Comment:
This is the final page and should have some sort of recap of what has been covered, as well as some links to other things of interest.

The 'Finding Help' idea is a good one, but on it's own seems a little negative.  Have a look at https://tutorials.ubuntu.com/tutorial/snap-python-app#4